src/gatsby-theme-carbon/components/Layout.js

```diff
@@ -32,7 +24,6 @@ const Layout = ({
   tabs,
 }) => {
   const is404 = children.key === null;
-  const [isBannerVisible, setBannerVisibility] = useState(true);
```

Comment:
Do we perhaps just want to set this to false? That way when we want to reenable the banner we can just set this back to true // a different local storage item

Reply:
@tw15egan I feel a bit better about just removing it all together so we don't have unused ghost styles and code. Like you said, we'll want to revisit our local storage approach next time and that makes up the bulk of the component.

We can always come back and reference this implementation if needed.